### ISSUE TITLE
Notifications - page and full text

### ISSUE THREAD

**Issue by dapgo** [enaon/ninebot-one-nRF52#5]
Feature:
Currently the notification read only the 1st BLE command; so notification is limited to 1st 12 characters, what is alright for the notification area on "Main" screen.

However it would be good to have a notification page where you can see the full text of received notifications.
So the BLE packets should be read and parsed until  /r/n is received.

**Comment by enaon** [comment]
Yes, I will add that, I was out of flash space untill recently, so it made little sence to really support notifications, there was no flash or ram space left for them. Thanks to fanoush we now have 40K more, so yes I will support them in their own face and files, more or less the way you recomented on the other issues. Thanks :)

**Comment by enaon** [comment]
this is now supported, use the push face to view and delete info/note/todo entries.
